nautilus_trader/adapters/bybit/data.py

```diff
@@ -131,22 +133,24 @@ def __init__(
 
         # WebSocket API
         self._ws_clients: dict[BybitInstrumentType, BybitWebsocketClient] = {}
+        self._decoders: dict[str, dict[BybitInstrumentType, msgspec.json.Decoder]] = defaultdict(
+            dict,
+        )
         for instrument_type in instrument_types:
             self._ws_clients[instrument_type] = BybitWebsocketClient(
                 clock=clock,
-                handler=lambda x: self._handle_ws_message(instrument_type, x),
+                handler=partial(self._handle_ws_message, instrument_type),
```

Comment:
strangely, this partial is needed instead of the lambda when subscribing to both spot and linear instruments. to me, not clear why. @cjdsellers  any ideas?

Reply:
with the lambda, only 1 instrument type ticks are created.